models/build.js

```diff
@@ -100,6 +100,18 @@ const MODEL = {
         .description('Status message to describe status of the build')
         .example('Build failed due to infrastructure error'),
 
+    stats: Joi
```

Comment:
can the keys be dynamic ? If tomorrow we want to track another build stat, can we do that without a code change ?

Reply:
For example if custom executors want to track metrics specific to them, say executor iOS one wanting to track VM setup time, executor  hyperd tracking VM creation time.

Reply:
yeah we can make it just `object`

Reply:
Use `unknown` option instead so that for defined fields like `queueTime` it will check that the it's `isoDate`. This option will still allow adding other fields.